flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/config/OperatorConfigOptions.java

```diff
@@ -33,7 +33,7 @@
                     .durationType()
                     .defaultValue(Duration.ofSeconds(60))
                     .withDescription(
-                            "The interval for the controller to reschedule the reconcile process");
+                            "The interval for the controller to reschedule the reconcile process.");
```

Comment:
I have updated this in #141.

Reply:
Thanks for the information. As we do not automatically generate documents for now, I will roll back this change and only keep it in our doc website.